### DISCUSSION TITLE
How to get hands on sleeve and open sourcerer badges??

### DISCUSSION THREAD

**Discussion by SHVM-09** [community/community#70518] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have almost got all badges except those two , Also process for such isn't available clearly! so please help me out here!

**Comment by ByMykel** [comment]
Those two badges are being tested, so for now they are not public. GitHub probably will do a announcement like [this](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/) when they are out.

**Reply by SHVM-09** [reply]
Thanks for response !!

**Comment by itsdotnet** [comment]
The 'Heart On Your Sleeve' & 'Open Sourcer' achievement badges are not yet released to public GitHub users at the moment.

Please kindly wait for & stay tuned to the official news release by GitHub on their blog over [here](https://github.blog/).

When they launched the first set of achievement badges, they previously also posted a blog over [here](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/) to let users know more information. Hence, they might do the same thing (i.e. post a new blog) to announce updates to the achievement system on GitHub.

Other than that, there has been no information (based on my understanding) on the mechanics to achieve those two new badges.

P.s: I am also interested in collect badges so I am also researching. Now its impossible to get this 2 badges.

**Comment by ygzdvr** [comment]
The 'Heart On Your Sleeve' and 'Open Sourcerer' badges you mentioned are currently like rare treasures yet to be discovered by the public. GitHub is probably in the process of fine-tuning these badges before they release them into the system.

When the time is right, GitHub will most likely shock us with a sparkling announcement on their blog, similar to the grand reveal they did when the first series of achievement badges made their debut.

In the meantime, all we can do is keep our eyes on GitHub’s blog for any new releases that will explain on how we can get our hands on these exclusive badges.

**Reply by SHVM-09** [reply]
Thanks for response !!

**Reply by abhijeetvaghela** [reply]
any update on this?

**Comment by 0xNunana** [comment]
As of the time of answering, the badge you're seeking to have is still being tested hence its unavailable. The procedure to obtain it will be communicated when its released to the public

**Reply by SHVM-09** [reply]
Thanks for response !!

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @SHVM-09. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Reply by ousloob** [reply]
@github-staff 
This is unfortunate news. Perhaps instead of removing Achievements entirely from the Community, there could be a way to refine them to encourage meaningful contributions. For example, an "Open Sourcer" achievement could be earned by maintaining an open-source project with a certain number of users or contributions. This way, Achievements would continue to inspire positive and valuable engagement rather than low-quality posts.

Would love to hear if there are any future plans to bring back Achievements in a more constructive way! 🚀

**Reply by nejlontacsko** [reply]
I support the idea proposed by @ousloob !!
@github-staff Can you see an opportunity to implement it?

**Comment by nylla8444** [comment]
just saw this earlier! boy it was beautifull

<img width="551" height="576" alt="image" src="https://github.com/user-attachments/assets/9b9a948e-b963-4268-9a02-28d587445b97" />

**Reply by itsmanudon** [reply]
Did you make any open source contributions? Which were accepted

**Reply by nylla8444** [reply]
yes

but after few 1hour+ github decided to remove it tho.
My achievements now:
<img width="326" height="144" alt="image" src="https://github.com/user-attachments/assets/1511053e-af23-496a-9016-4e390cb88990" />

**Reply by ajm19826** [reply]
@github-staff 
why were they removed again?

**Reply by nylla8444** [reply]
I want to know the reason too

haha maybe intern just accidentally pushed it to prod?

**Reply by Mr-the-beginner** [reply]
mine got removed too

**Comment by samus-aran** [comment, marked as answer]
We apologize for the temporary re-instatement of the **Heart On Your Sleeve** and **Open Sourcerer** achievements. This was done in error - those achievements were part of an experimental rollout and weren’t meant to be re-enabled broadly.

Once we identified the mistake, we removed them again.

We’re sorry if this was disappointing. Thanks for bearing with us, and we really appreciate your patience and understanding.

**Reply by ajm19826** [reply]
@samus-aran when will the achievements be rolling out?

**Reply by Mr-the-beginner** [reply]
oh cmon, thats awful

**Comment by marco-crest** [comment]
good day @github-staff!
any updates on how to get hands on sleeve and open sourcerer badges??
the only 2 missing for me (apart form the mars rover badge which is impossible to get atm)
thank you
